dropwizard-testing/src/main/java/io/dropwizard/testing/junit/ResourceTestRule.java

```diff
@@ -37,11 +38,14 @@
      */
     public static class Builder {
 
+        private static Consumer<ClientConfig> NOOP_CONSUMER = c -> {};
```

Comment:
Sounds good. Too bad Java doesn't provide something like `Consumer.noop()` akin to `Function.identity()`.